pipeline/acquisition.py

```diff
@@ -0,0 +1,390 @@
+    """
+
+    class Experimenter(dj.Part):
+        definition = """
+        -> master
+        -> reference.Experimenter
+        """
+
+    class ExperimentType(dj.Part):
+        definition = """
+        -> master
+        -> ExperimentType
+        """
+
+
+@schema
+class BehaviorAcquisition(dj.Imported):
+    definition = """
+    -> Session
+    """    
+    
+    class LickTrace(dj.Part):
+        definition = """
+        -> master
+        ---
+        lick_trace_left: longblob   
+        lick_trace_right: longblob
+        lick_trace_start_time: float # first timepoint of lick trace recording
+        lick_trace_sampling_rate: float # sampling rate of lick trace recording
+        """       
+        
+    def make(self,key):
+        ############## Dataset #################
+        sess_data_dir = os.path.join('..','data','whole_cell_nwb2.0')
+                
+        # Get the Session definition from the keys of this session
+        animal_id = key['subject_id']
+        date_of_experiment = key['session_time']
+        
+        # Search the files in filenames to find a match for "this" session (based on key)
+        sess_data_file = utilities.find_session_matched_nwbfile(sess_data_dir, animal_id, date_of_experiment)
+        if sess_data_file is None: 
+            return
+        nwb = h5.File(os.path.join(sess_data_dir,sess_data_file), 'r')
+        
+        #  ============= Now read the data and start ingesting =============
+        self.insert1(key)
+        print('Insert behavioral data for: subject: {0} - date: {1}'.format(key['subject_id'],key['session_time']))
+        # -- MembranePotential
+        key['lick_trace_left'] = nwb['acquisition']['timeseries']['lick_trace_L']['data'].value
+        key['lick_trace_right'] = nwb['acquisition']['timeseries']['lick_trace_R']['data'].value
+        lick_trace_time_stamps = nwb['acquisition']['timeseries']['lick_trace_R']['timestamps'].value
+        key['lick_trace_start_time'] = lick_trace_time_stamps[0]
+        key['lick_trace_sampling_rate'] = 1/np.mean(np.diff(lick_trace_time_stamps))        
+        self.LickTrace.insert1(key, ignore_extra_fields=True)
+
+
+@schema
+class PhotoStimulation(dj.Manual):
+    definition = """ # Table containing information relating to the stimulatiom (stimulation type (optical or electrical), location, device)
+    -> Session
+    photostim_datetime: varchar(36) # the time of performing this stimulation with respect to start time of the session, in the scenario of multiple stimulations per session
+    ---
+    -> stimulation.PhotoStimulationInfo
+    photostim_timeseries: longblob
+    photostim_start_time: float # first timepoint of photostim recording
+    photostim_sampling_rate: float # sampling rate of photostim recording
+    """    
+
+
+@schema
+class Cell(dj.Manual):
+    definition = """ # Information relating to the Cell and the intracellular recording of this cell (e.g. location, recording device)
```

Comment:
Again, the comment should define what each element of the entity set represents.